Description: Get frobby version.
 Since the Debian frobby package doesn't include stdinc.h, which contains
 frobby's version, we copy the necessary code over.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2015-06-04

--- a/M2/Macaulay2/d/version.dd
+++ b/M2/Macaulay2/d/version.dd
@@ -38,7 +38,9 @@
    extern char timestamp[];
    #undef ASSERT
    #ifdef HAVE_FROBBY
-     #include <stdinc.h>		// to get version number of frobby
+     namespace constants {
+       extern const char* const version;
+     }
      #define FROBBY_VERSION constants::version
    #else
      #define FROBBY_VERSION \"not present\"
